vector/src/main/AndroidManifest.xml

```diff
@@ -75,7 +75,7 @@
                 android:value=".features.home.HomeActivity" />
         </activity>
         <activity android:name=".features.debug.DebugMenuActivity" />
-        <activity android:name=".features.home.createdirect.CreateDirectRoomActivity" />
+        <activity android:name="im.vector.riotx.features.createdirect.CreateDirectRoomActivity" />
```

Comment:
Why this change?

Reply:
I think we should split the home package as it's not a "feature". But as you want

Reply:
I was talking about the fact that `im.vector.riotx` has been added. But ok for the move to another package.